src/config.cpp

```diff
@@ -63,7 +63,12 @@ struct SessionOptions_Element : JSON::Element {
     if (name == "log_id")
       v_.log_id = value;
     else if (name == "enable_profiling")
+#ifdef _WIN32
```

Comment:
#ifdef _WIN32 everywhere is what we wanted to avoid with std::filesystem. What problem is this change solving?

Reply:
Saw the other comment, hopefully you can put the #ifdef _WIN32 inside the "oga_std_filesystem" replacement so that we can switch to it once the compilers sort themselves out.